### PR TITLE
button opacity layer border on button press

### LINKED ISSUE #1102: Opacity layer not following borderRadius on button press
Hi guys, 

Sorry I'm opening this again, I know this issue has appeared a lot here, I have read #328, #535, #1035, #581, however I cannot make it work..

### Description

1. I have a button with borderRadius 

```
<Button 
        title='LOG IN'
        style={{ borderRadius: 50 }}
        borderRadius={25}
        buttonStyle={{ 
               width: 260,
               backgroundColor: '#58D7B5',
               borderWidth: 1,
               borderColor: '#fff',
               alignSelf: 'center',
               margin: 16,
               elevation: 0, 
               borderRadius: 25 }}
         containerViewStyle={{borderRadius:25}}
         // containerStyle={ {elevation: 0, borderRadius: 25}}
 />
```
2. I am expecting to see the border radius applied to the opacity layer when pressing the button.
3. But is just a rectangle instead.

button_bug

I am using "react-native-elements": "^1.0.0-beta4"

### Reproduction Steps and Sample Code

Snack example:

https://snack.expo.io/HyUL9T3if

Any help would be much appreciate 🙏
Thanks!

### PR DESCRIPTION
**Pull request by Gregjarvez**
fixes #1102

### CODE REVIEW

**Comment by iRoachie** [comment]
Wrong base :P

**Comment by Gregjarvez** [comment]
thank you !

File: src/buttons/Button.js [hunk 1]
```diff
@@ -56,6 +56,16 @@ class Button extends Component {
       ...attributes
     } = this.props;
 
+    if (Platform.OS === 'android' && (borderRadius && !attributes.background)) {
```
Comment (xavier-villelegier) [inline src/buttons/Button.js]: `borderRadius` doesn't exist here. You need to use `containerStyle.borderRadius`

Comment (Gregjarvez) [reply src/buttons/Button.js]: when you rely on linters too much XD

File: src/buttons/Button.js [hunk 2]
```diff
@@ -56,6 +56,19 @@ class Button extends Component {
       ...attributes
     } = this.props;
 
+    if (
+      Platform.OS === 'android' &&
+      (containerStyle.borderRadius && !attributes.background)
```
Comment (xavier-villelegier) [inline src/buttons/Button.js]: Mmh my bad I think it's `buttonStyle.borderRadius`

**Pull request merged**